### PR TITLE
Minor proofreading of EIP-2938 for readability

### PR DESCRIPTION
**Pull request by lrettig**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 2938 requires approval from one of (@samwilsn, @villanuevawill, @lightclient, @vbuterin, @adietrichs)

**Review by lightclient — COMMENTED** [review]
Sorry we missed this. These changes look good to me.

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - eip-2938.md requires approval from one of (vbuterin, adietrichs, lightclient, villanuevawill, samwilsn)
 @vbuterin @adietrichs @lightclient @villanuevawill @samwilsn

**Comment by lrettig** [comment]
Cool. @lightclient I think this is still waiting on approval. Anything else I need to do here first?

File: EIPS/eip-2938.md
Comment (SamWilsn) [inline EIPS/eip-2938.md:165]: I believe this changes the meaning of the sentence unintentionally.

**Review by SamWilsn — CHANGES_REQUESTED** [review]

File: EIPS/eip-2938.md
Comment (lrettig) [reply EIPS/eip-2938.md:165]: Will reword to make this clearer, good catch

**Comment by axic** [comment]
@SamWilsn @lightclient can you review again?

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**
